lab-teddy/lib/collecting-data.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = function(callback){
+  let dataArray = [];
+  fs.readFile(`${__dirname}/../data/one.text`, function(err, data){
+    if(err) throw err;
```

Comment:
generally, in lib helper functions, you will want to pass errors into your callbacks, instead of throwing it. if(err) callback(err);